### PR TITLE
Fdg 7675Research Spending & Revenue chart animation implementation

### PR DESCRIPTION
**Pull request by chasls2**
[FDG-7675](https://federal-spending-transparency.atlassian.net/browse/FDG-7675)

Animation across the graph

### CODE REVIEW

**Review by wyattch — CHANGES_REQUESTED** [review]
Had a few comments, everything else looks good

File: src/layouts/experimental/charts/lineGraph.jsx [hunk 1]
```diff
@@ -1,118 +1,174 @@
...
+                       {point.serieId} : {point.data.y}
+                    </div>
+                  ))}
+                </div>
+              </div>
+            </div>
+          );
+        }}
+      />
+      {/* {mouseHover === null && (
```
[the remark this answers is not in the document]
Comment (chasls2) [reply src/layouts/experimental/charts/lineGraph.jsx:159]: I forgot all about all the lineGraph stuff I didn't revert

File: src/layouts/experimental/charts/lineGraph.jsx [hunk 2]
```diff
@@ -1,118 +1,174 @@
...
+          legendOffset: -40,
+          legendPosition: 'middle',
+        }}
+        animate={true}
+        enableGridX={false}
+        enableGridY={true}
+        enablePoints={false}
+        crosshairType='x'
+        onMouseEnter={(slice) => {
+          console.log('Enter: ', setMouseHover(slice.slicePoints[0].x));
```
[the remark this answers is not in the document]
Comment (chasls2) [reply src/layouts/experimental/charts/lineGraph.jsx:109]: user currently aren't using this file. My bad I kind of just forgot about it

**Review by wyattch — APPROVED** [review]
Approved

**Pull request merged**